fabric-connector/src/main/java/com/luxoft/fabric/FabricConnector.java

```diff
@@ -17,128 +21,63 @@
  */
 public class FabricConnector {
 
-    public static class Options {
-        EventTracker eventTracker;
-
-        public EventTracker getEventTracker() {
-            return eventTracker;
-        }
-
-        public Options setEventTracker(EventTracker eventTracker) {
-            this.eventTracker = eventTracker;
-            return this;
-        }
-    }
-
-    private final Logger logger = LoggerFactory.getLogger(this.getClass());
-
-    protected HFClient hfClient;
-    protected FabricConfig fabricConfig;
+    private static final Logger LOGGER = LoggerFactory.getLogger(FabricConnector.class);
```

Comment:
Can you, please, avoid renaming `logger`? I understand that you add some expressiveness here, but I cannot see it, sorry. For me all caps means, it is a scalar-like constant  (BigInteger.ZERO, Config.DEFAULT_CONFIG_FILE), or a magic keyword (C/C++ like macro).

Reply:
Sorry for this. 

That was my habit introduced by previous usage of checkstyle, which considers all private static final fields as constants